content/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Organic Maps: Offline Hike, Bike, Trails and Navigation"
-description: Fast detailed offline maps for travelers, tourists, hikers and cyclists, based on OpenStreetMap and curated with love by MapsWithMe founders.
+description: Fast detailed offline maps for travelers, tourists, hikers and cyclists, based on OpenStreetMap and curated with love by MapsWithMe (MapsMe) founders.
```

Comment:
MAPS.ME is a registered trademark.

Reply:
MapsMe too? Where did you check?

Reply:
Am I breaking trademark rules if my CV shows that I am a founder and used to work in MAPSME?